README.md

```diff
@@ -483,3 +511,66 @@ recomputed tree head 043bc6b0e49a085f2370b2e0f0876d154c2e8d8fe049077dbad118a3635
 authentication path
  8964f010faa9e499b21917f8792b541b7b1ac19f313a5d53094c698c2edc330b
 ```
+
+### Mirroring a CA
+
+We can set up a new mirror with the `mtc mirror new` command:
+
+```
+$ mtc mirror new ca.example.com/path
+```
+
+This will download the `ca-params`
+from `https://ca.example.com/path/mtc/v04b/ca-params` and
+set up a directory structure similar to that of a CA:
+
+```
+$ find .
+.
+./www
+./www/mtc
+./www/mtc/v04b
+./www/mtc/v04b/ca-params
+./www/mtc/v04b/batches
+./tmp
+```
+
+To bring the mirror up to date with the CA, use the `update` command:
```

Comment:
We should add a server mode for the mirror too, so that it periodically updates from the CA and serves the files over HTTP.

Reply:
Indeed: #58